lib/commands.js

```diff
@@ -637,7 +637,7 @@ exports.bindCommands = function bindCommands(forum) {
             args.shift();
             commandText = args.shift();
             mention = true;
-        } else if (/^!\S{3,}(\s|$)/.test(line)) {
+        } else if (/^!\w\S{2,}(\s|$)/.test(line)) {
```

Comment:
Well, that's the question, isn't it? It's a breaking change: commands must start with an alphanumeric-or-underscore character, but can contain anything after that, making them equivilent to identifiers in several programming languages. The pattern wasn't unicode-aware to begin with, and the \P syntax wasn't added to es6.